test/mosaic-integration-tests/src/test/java/org/eclipse/mosaic/test/HighwayReleaseIT.java

```diff
@@ -42,4 +43,30 @@ public void executionSuccessful() {
         assertNull(simulationResult.exception);
         assertTrue(simulationResult.success);
     }
+
+    /**
+     * This checks whether a cell message takes the expected amount to be send from tmc to vehicle
+     * and from vehicle to tmc.
+     * Should be 50ms for tmc uplink + 200ms for vehicle downlink + 200ms for vehicle uplink + 50ms for tmc downlink.
+     */
+    @Test
+    public void roundTripMessageTakesRightAmountOfTime() throws Exception {
```

Comment:
@kschrab is this test sufficient? I used simple `ConstantDelay`s and I think I over-complicated the apps a little bit

Reply:
so the complete round trip requires 500ms? 

btw, better use `10 * TIME.SECOND + 250 * TIME.MILLSECOND` instead of 10250000000L and add a comment which delays from the server_config.json are used when

Reply:
Made the test more comprehensible.